zetaclient/chains/sui/client/client_live_test.go

```diff
@@ -183,6 +184,19 @@ func TestClientLive(t *testing.T) {
 		require.NotNil(t, res)
 		require.Equal(t, TxStatusFailure, res.Effects.Status.Status)
 	})
+
+	t.Run("GetObjectParsedData", func(t *testing.T) {
```

Comment:
Maybe we can have a test with an invalid object id?

Reply:
https://github.com/zeta-chain/node/pull/3850/commits/4bdf29bd602d81ff24b5577c12d9ada419f6975b